LayoutTests/svg/dom/SVGAnimationElement-begin-end-arguments.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<script src="../../resources/js-test.js"></script>
+<svg>
+  <animate></animate>
+</svg>
+<script>
+description('Check that incorrect number of arguments throw TypeError.');
+
+var animateElm = document.querySelector('animate');
+
+debug('SVGAnimationElement');
+
+debug('');
+debug('beginElementAt(float offset)');
+shouldThrow('animateElm.beginElementAt()');
```

Comment:
Please add this case to the SVGAnimationElement-begin-end-arguments-typechecks.html and remove this test.